app/controllers/shipit/stacks_controller.rb

```diff
@@ -39,6 +39,11 @@ def destroy
     end
 
     def settings
+      client = Shipit::ApiClient.find_by(creator: current_user, name: "#{current_user.name} ccmenu client")
+      client ||= Shipit::ApiClient.create!(
```

Comment:
Yeah permission (for the worse or the better) is a serialized field, as such it can't be used for querying, only inserting. And `find_or_create_by()` uses the arguments for both querying and inserting.